mesonbuild/scripts/meson_exe.py

```diff
@@ -78,6 +78,11 @@ def run_exe(exe):
                          stdout=subprocess.PIPE,
                          stderr=subprocess.PIPE)
     stdout, stderr = p.communicate()
+
+    if p.returncode == 3221225781:
```

Comment:
3221225781 == 0xc0000135, which is much easier to look up in headers as `STATUS_DLL_NOT_FOUND`.